packages/react-scripts/template/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "React App",
+  "icons": [
+    {
+      "src": "favicon.ico",
+      "sizes": "192x192",
+      "type": "image/png"
```

Comment:
Yes, my understanding is that `type` and `sizes` are required: https://developers.google.com/web/updates/2015/10/splashscreen

While there is a good amount of overlap between metadata that's in `index.html` and what's in this `manifest.json` file, there are things that you couldn't trivially deduce—like the `background_color`, the `sizes` of the icon, or the distinction between the `short_name` and `name`.

I'm a little hesitant to go down the route of requiring developers to partially populate `manifest.json` and then infer the rest of the fields from `index.html`, as that feels a bit too magic to me.

I'll definitely add some documentation explaining `manifest.json` in general, though.